src/gen/templates/licenseInfo.mustache

```diff
@@ -0,0 +1,11 @@
+/**
+ * {{{appName}}}
+ * {{{appDescription}}}
+ *
+ * {{#version}}OpenAPI spec version: {{{version}}}{{/version}}
+ * {{#infoEmail}}Contact: {{{infoEmail}}}{{/infoEmail}}
+ *
+ * NOTE: This file is auto generated by the swagger code generator program.
```

Comment:
Let's consider changing this line. Let's discuss.

Reply:
Changed to "This file is auto generated do not edit the file manually."